fontforge/splinesaveafm.c

```diff
@@ -534,13 +534,13 @@ return( 0 );
 	fclose(file);
 return( 0 );
     }
-    tfmd.kerntab = calloc(tfmd.kern_size,sizeof(uint8));
-    tfmd.ligkerntab = calloc(tfmd.ligkern_size,sizeof(uint8));
-    tfmd.ext = calloc(tfmd.esize,sizeof(uint8));
-    tfmd.ictab = calloc(tfmd.italic_size,sizeof(uint8));
-    tfmd.dptab = calloc(tfmd.depth_size,sizeof(uint8));
-    tfmd.httab = calloc(tfmd.height_size,sizeof(uint8));
-    tfmd.widtab = calloc(tfmd.width_size,sizeof(uint8));
+    tfmd.kerntab = calloc(tfmd.kern_size,4*sizeof(uint8));
```

Comment:
Hmm -- that's the second `calloc` argument rather than the first. But I guess I have no basis for claiming the first argument is more correct.